.github/workflows/release.yaml

```diff
@@ -0,0 +1,113 @@
+name: release
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*"
+
+jobs:
+  build-docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout code
+        uses: actions/checkout@v2.3.5
+
+      - run: |
+          echo "GIT_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      -
+        name: Docker meta
+        id: docker_meta
+        uses: docker/metadata-action@v3.5.0
+        with:
+          images: |
+            ghcr.io/tus/tusd
+            tusproject/tusd
+          flavor: |
+            latest=${{ startsWith(github.ref, 'refs/tags/') }}
```

Comment:
Does this set the `latest` tag for the current image?

Reply:
One more thing: We would like to only set the latest tag if the tag is no pre-release (i.e. not of the form `v1.2.3-pre1`). In that case, the tags v1 and v1.2 should also not be set. Is that easily possible or should we use an entirely different tag format for pre-releases?

Reply:
If you check the runs I did, under the step Docker meta, and the Docker tags section, you see what tags it generates

`v300.0.0-pre1`
![image](https://user-images.githubusercontent.com/1681525/137949446-34b9d185-61cb-40f1-ac04-fa84d2f98462.png)

`v200.1.2`
![image](https://user-images.githubusercontent.com/1681525/137949527-a0a370ea-0d99-40d9-9d5a-9626638eae12.png)

`push to master` (in the test run it's to branch called `local-testing` for a quick test)
![image](https://user-images.githubusercontent.com/1681525/137949642-b545367d-be9f-41f9-8ca7-6a05c0ae5be7.png)